components/footer/Footer.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  FAUCETS,
+  FooterLinkInfo,
+  LEGAL,
+  NETWORKS,
+  RESOURCES,
+  SDKs,
+  SOLUTIONS,
+} from "./footerLinks";
+import { SOCIALS } from "./socialLinks";
+import {
+  Box,
+  ButtonGroup,
+  Container,
+  Flex,
+  SimpleGrid,
+  Stack,
+} from "@chakra-ui/react";
+import { Logo } from "components/logo";
+import { PRODUCTS } from "components/product-pages/common/nav/DesktopMenu";
+import {
+  Heading,
+  LinkButton,
+  TrackedIconButton,
+  TrackedLink,
+} from "tw-components";
+
+type FooterLinkGroupProps = {
+  heading: string;
+  links: FooterLinkInfo[];
+};
+
+function FooterLinkGroup(props: FooterLinkGroupProps) {
+  return (
+    <Stack spacing="4" minW="36" flex="1" mb={{ md: 12, base: 14 }}>
+      <Heading as="h5" size="label.lg">
+        {props.heading}
+      </Heading>
+      <Stack spacing="3" shouldWrapChildren>
+        {props.links.map((link) => (
+          <TrackedLink
+            isExternal={link.isExternal}
+            key={link.label}
+            href={link.link}
+            category="footer"
+            label={link.label}
+          >
+            {link.name}
+          </TrackedLink>
+        ))}
+      </Stack>
+    </Stack>
+  );
+}
+
+function FooterLinksGrid() {
+  return (
+    <Flex
+      mb={10}
+      direction={{ base: "column-reverse", md: "column", lg: "row" }}
+      gap={{ base: "12", md: "8" }}
+    >
```

Comment:
what does this <Flex> do since it just contains 1 component now?

Reply:
@jnsdls I was also wondering the same - but because it was [already there](https://github.com/thirdweb-dev/dashboard/blob/main/components/product-pages/homepage/Footer.tsx#L109-L111) I did not remove it. It contained 1 component earlier as well ('SimpleGrid') - I am removing it now

Reply:
Removed it in https://github.com/thirdweb-dev/dashboard/pull/716/commits/8e9a2adb8f2ec1b7d3a8787d56fdab718371f127